.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug report
+about: Create an issue to report a bug for the SAM Translator
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- Make sure we don't have an existing Issue that reports the bug you are seeing (both open and closed). 
+If you do find an existing Issue, re-open or add a comment to that Issue instead of creating a new one. -->
+
+### Description:
+<!-- Briefly describe the bug you are facing.-->
+
+
+
+### Steps to reproduce:
+<!-- Provide detailed steps to replicate the bug, including steps from third party tools (CDK, etc.) -->
+
+
+
+### Observed result:
+<!-- Please provide command output with `--debug` flag set.-->
+
+
+
+### Expected result:
+<!-- Describe what you expected.-->
+
+
+
+### Additional environment details (Ex: Windows, Mac, Amazon Linux etc)
+
+1. OS:
+2. `sam --version`:
```

Comment:
is sam-cli required here?

Reply:
Maybe add a disclaimer? (If using SAM through the CLI):
Seems a little bit verbose.